pkg/status/render/fixtures/check_stats.text

```diff
@@ -25,6 +26,7 @@ Collector
     disk
     ----
       Instance ID: disk [OK]
+      Long Running Check: false
```

Comment:
Should we add a test case that exercises the case of a long-running check?

Reply:
Done with `cilium` check.